cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,51 @@
+/// <reference types='cypress' />
+
+const { ar } = require("@faker-js/faker");
+
 describe('', () => {
-  before(() => {
+  let user;
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.login(user.email, user.username, user.password);
+      cy.visit('/');
+    });
   });
 
-  it('', () => {
-
+  it('should have the ability to create an article', () => {
+    let article;
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+      cy.get('.ion-compose').click();
+      cy.get('[placeholder="Article Title"]').type(article.title);
+      cy.get('[placeholder="What\'s this article about?"]').type(article.about);
+      cy.get('[placeholder="Write your article (in markdown)"]')
+      .type(article.body);
+      cy.get('button.btn-primary').contains('Publish Article').click();
+      cy.get('button.btn-outline-danger').eq(0).should('exist');
+      cy.get('button.btn-outline-danger').eq(0).should('be.enabled');
+      cy.get('button.btn-outline-danger').eq(1).should('exist');
+      cy.get('button.btn-outline-danger').eq(1).should('be.enabled');
```

Comment:
the best idea here, is to check article title. You assertion could be as an additional checks